package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-core-ui",
-    "version": "1.20.3",
+    "version": "1.20.4",
```

Comment:
regardless that is only FE, please bump version at `manifest` file;

Reply:
> regardless that is only FE, please bump version at `manifest` file;

There is no `manifest` in this repo, I'll bump it in [related PR in tao-core](https://github.com/oat-sa/tao-core/pull/2769)